store/postgres_migration_test.go

```diff
@@ -1,151 +1,147 @@
 package store
 
 import (
-	"database/sql"
+	"context"
 	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
-	"github.com/valocode/bubbly/config"
 	"github.com/valocode/bubbly/env"
-
-	"github.com/ory/dockertest/v3"
-	"github.com/ory/dockertest/v3/docker"
-	"github.com/valocode/bubbly/api/core"
-
-	"github.com/stretchr/testify/require"
+	"github.com/valocode/bubbly/test"
```

Comment:
❔ This is more of a question, but I think both of our beloved IDEs have an option to group "our" modules together and separate from "theirs". Do we want that?